src/get-started/install/_get-sdk-win_v2.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Windows install
+description: How to install on Windows v2.
+short-title: Windows
+next:
+  title: Set up an editor
+  path: /get-started/windows-quick
+---
+
+## Get the Flutter SDK
+
+
```

Comment:
**suggestion:** If there are two ways to do this, consider making this a tabbed interface.